lib/ip.go

```diff
@@ -57,6 +101,39 @@ func RandIP6() net.IP {
 	return net.IP(ip[:])
 }
 
+// RandIP6Range returns a list of randomly generated IPv6 addresses within
+// starting and ending IPrange.
+func RandIP6Range(StartIP, EndIP string) (net.IP, error) {
```

Comment:
Please try to apply all the same fixes I said above for IPv4, over here as well.